### PR TITLE
Travis stages

### PR DESCRIPTION
**Pull request by rochoa**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,9 +16,9 @@
     "mocha": "^3.5.0"
   },
   "scripts": {
-    "test": "mocha && npm run lint",
-    "test:watch": "mocha -w",
-    "lint": "$(npm bin)/jshint src/ test/"
+    "test": "mocha",
+    "test:watch": "mocha -w -R dot",
+    "lint": "jshint src/ test/"
```
Comment (IagoLast) [inline package.json]: does this works when jshint is not installed globally?

Comment (rochoa) [reply package.json]: Yes. See https://docs.npmjs.com/misc/scripts#path.

BTW, it's the same as in "mocha", isn't it?

Comment (IagoLast) [reply package.json]: right! good to know

**Comment by dv343** [comment]
I think we can close this now: mocha, travis and linting are already configured. See this for example: https://travis-ci.org/CartoDB/tangram-cartocss/builds/301408864?utm_source=github_status&utm_medium=notification

**Pull request closed without merging**